baseplate/__init__.py

```diff
@@ -302,103 +295,7 @@ def register(self, observer: BaseplateObserver) -> None:
         """
         self.observers.append(observer)
 
-    def configure_logging(self) -> None:
-        """Add request context to the logging system.
-
-        .. deprecated:: 1.0
-
-            Use :py:meth:`configure_observers` instead.
-
-        """
-        # pylint: disable=cyclic-import
-        from baseplate.observers.logging import LoggingBaseplateObserver
-
-        self.register(LoggingBaseplateObserver())
-
-    def configure_tagged_metrics(self, metrics_client: metrics.Client) -> None:
-        """Register a Tagged Metrics Observer on the metrics client.
-
-        Only called if already determined that the user has configured tagging to be on.
-
-        :param metrics_client: Metrics client to send request metrics to.
-        :param app_config: The configuration file for the application to parse for metrics tags and flags
-
-        """
-        # pylint: disable=cyclic-import
-        from baseplate.observers.metrics_tagged import TaggedMetricsBaseplateObserver
-
-        self._metrics_client = metrics_client
-        self.register(
-            TaggedMetricsBaseplateObserver.from_config_and_client(self._app_config, metrics_client)
-        )
-
-    def configure_metrics(self, metrics_client: metrics.Client) -> None:
```

Comment:
`configure_metrics` is referred to in `serve.rst`

Reply:
eek, thanks! fixed